data/achievements.cfg

```diff
@@ -1,18 +1,43 @@
 #textdomain wesnoth
 
-#define ACHIEVEMENT CONTENT_FOR ID NAME DESC
+#define ACHIEVEMENT ID NAME DESC
 #arg HIDDEN
 no#endarg
 
+#arg NAME_COMPLETE
+#endarg
+
+#arg DESC_COMPLETE
+#endarg
+
+#arg DATA_ICON
+#endarg
+
+#arg DATA_ICON_COMPLETE
+#endarg
+
     [achievement]
-        content_for={CONTENT_FOR}
         id={ID}
         name={NAME}
+        name_completed={NAME_COMPLETE}
         description={DESC}
+        description_completed={DESC_COMPLETE}
+        data_uri_icon={DATA_ICON}
+        data_uri_icon_completed={DATA_ICON_COMPLETE}
```

Comment:
As mentioned before, I don't like encoding the format into the key, as that limits us from expanding it later. I'd prefer just `icon` and `icon_completed` even if they truly are limited to DataURIs only.

Reply:
Testing it out, it looks like it's plain incorrect anyway since full paths (ie `data/add-ons/ANLEra/images/halo/shadow-mage-halo6.png` or `data/core/images/icons/bag.png`) do also work as-is.

Reply:
Yeah, I suspected that would already work.

Reply:
Updated to just `icon` and `icon_completed`.